lib/services/platform-service.ts

```diff
@@ -256,7 +256,11 @@ export class PlatformService extends EventEmitter implements IPlatformService {
 		return true;
 	}
 
-	public async validateOptions(provision: true | string, teamId: true | string, projectData: IProjectData, platform?: string): Promise<boolean> {
+	public async validateOptions(provision: true | string, teamId: true | string, projectData: IProjectData, platform?: string, aab?: boolean): Promise<boolean> {
+		if (platform && !this.$mobileHelper.isAndroidPlatform(platform) && aab) {
+			this.$errors.failWithoutHelp("Option --aab is supported for Android platform only.");
```

Comment:
I would recommend rephrasing this to "The --aab option is supported only for the Android platform" or "The --aab option is supported only for Android builds".